charts/gateway-operator/templates/cluster-role-1.6.yaml

```diff
@@ -319,6 +319,13 @@ rules:
     - patch
     - update
     - watch
+  - apiGroups:
+    - gateway-operator.konghq.com
+    resources:
+    - watchnamespacegrants
+    verbs:
+    - list
```

Comment:
Will we need `update/patch` permission of `watchnamespacegrants/status`?

Reply:
That resource does not have the status field currently: https://github.com/Kong/kubernetes-configuration/blob/75c0171148d9163e51520d5e16465694cc95a73f/api/gateway-operator/v1alpha1/watchnamespacegrant_types.go#L25-L26.

There's no reason to set it currently. If you see this as potentially helpful we could discuss this separately whether to add this field and what to fill it with.